dgraph/graphql/src/data.js

```diff
@@ -0,0 +1,18 @@
+import { RESTDataSource } from 'apollo-datasource-rest';
+
+export class dgraph extends RESTDataSource {
+  constructor() {
+    super();
+    this.baseURL = 'http://localhost:8080';
```

Comment:
at some other stage.. can we lift this string into a `.env` or CONSTANTS file or somewhere please?

Reply:
Yeah, good idea - we have a bunch that will need to eventually be moved out across both of our apis, will create a tidy up issue to track this.